appengine/bigquery/app.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2015 Google Inc. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+application: gae-bq-sample
```

Comment:
Is this still required for App Engine? I think you can specify it on the command line.